src/main/java/de/retest/recheck/ui/review/ReviewResult.java

```diff
@@ -31,6 +34,21 @@ public SuiteChangeSet createSuiteChangeSet( final String suiteName, final String
 		return nonEmpyChangeSets;
 	}
 
+	public List<ActionChangeSet> getAllActionChangeSets() {
+		return getSuiteChangeSets().stream() //
+				.flatMap( suiteChangeSet -> suiteChangeSet.getTestChangeSets().stream() ) //
+				.flatMap( testChangeSet -> testChangeSet.getAllActionChangeSets().stream() ) //
+				.collect( Collectors.toList() );
+	}
+
+	public List<AttributeDifference> getAttributeDifferences() {
+		return getAllActionChangeSets().stream()
+				.flatMap( actionChangeSet -> actionChangeSet.getAllAttributeChanges().stream()
+						.flatMap( attributeChanges -> attributeChanges.getChanges().values().parallelStream()
```

Comment:
Just like above: Don't nest these steps. And why a parallel stream?

```java
return getAllActionChangeSets().stream() //
		.flatMap( actionChangeSet -> actionChangeSet.getAllAttributeChanges().stream() ) //
		.flatMap( attributeChanges -> attributeChanges.getChanges().values().stream() ) //
		.flatMap( attributeDifferences -> attributeDifferences.stream() ) //
		.collect( Collectors.toList() );
```

This is more readable and, due to no nesting, each pipeline step is aligned.